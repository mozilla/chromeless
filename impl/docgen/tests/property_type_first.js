/** A module with a single property, and type before name */

/**
 * @property {integer} port
 * The port number of the URL, `null` if none was specified.
 */
