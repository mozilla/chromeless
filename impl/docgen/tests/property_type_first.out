{
  "desc": "A module with a single property, and type before name", 
  "filename": "tests/property_type_first.js", 
  "module": "property_type_first", 
  "properties": [
    {
      "desc": "The port number of the URL, `null` if none was specified.", 
      "type": "integer",
      "name": "port"
    }
  ]
}
